xls-workshop-openlane.ipynb

```diff
@@ -300,6 +300,16 @@
       "source": [
         "%%bash -c 'cat > user_module.x; interpreter_main user_module.x'\n",
         "\n",
+        "// The first line declares a function (fn) named \"adder1\" (Functions: https://google.github.io/xls/dslx_reference/#functions)\n",
```

Comment:
can you add the Japanese version too of those too?
https://github.com/chipsalliance/silicon-notebooks/blob/main/xls-workshop-openlane-ja.ipynb

Reply:
Yup, done with it.